packages/contracts/src/plugins/utils/Addresslist.sol

```diff
@@ -5,7 +5,7 @@ pragma solidity 0.8.17;
 import {CheckpointsUpgradeable} from "@openzeppelin/contracts-upgradeable/utils/CheckpointsUpgradeable.sol";
```

Comment:
Doesn't this contract have the same "base" category as for example MajorityVoting?  Two plugins inherit from it

Reply:
The parent folder `plugins/governance` would imply that this is a governance plugin. Hence, I put it into `plugins/utils`